app/controllers/application_controller.rb

```diff
@@ -125,15 +125,7 @@ def render_unauthorized
   end
 
   def saml_settings(options = {})
```

Comment:
Other than providing a default option for options, the method just seems to delegate to SAML::SettingsService could the wrapper go away?

Reply:
it totally could, I was just being lazy about rewriting specs that currently depend on this controller method, but by moving the logic in models, parts of session_controller could immedietely be refactored to use SAML::SettingsService.saml_settings(with_whatever_options).

Reply:
I can make that change to controllers and controller specs though if you prefer, but figured they'd naturally go away with future refactors.

Reply:
👍  either way works, LGTM 🚀